bodhi/server/consumers/signed.py

```diff
@@ -89,7 +89,9 @@ def __call__(self, message: fedora_messaging.api.Message):
                 log.info('Build is not assigned to release, skipping')
                 return
 
-            if build.update and build.update.from_tag:
+            if build.update and \
+               build.update.from_tag and not \
+               build.update.release.composed_by_bodhi:
```

Comment:
For readability I tend to put the and on the new line and I would encourage it even more for the ``not`` as it is pretty important :)

Reply:
updated :)